src/main/java/org/sagebionetworks/web/client/widget/header/Header.java

```diff
@@ -182,20 +184,30 @@ public void onFailure(Throwable caught) {
 
 	@EventHandler
 	public void onDownloadListUpdatedEvent(DownloadListUpdatedEvent event) {
-		// update Download List count, show if > 0
```

Comment:
This is a terrible inline diff.  Do what it used to do if not in alpha mode, otherwise make the DownloadListV2 component visible (the shopping cart icon with a number of how many files are in the cart).